test/uk/gov/hmrc/brm/models/PayloadSpec.scala

```diff
@@ -204,97 +118,48 @@ class PayloadSpec extends UnitSpec with WithFakeApplication {
       }
 
       "return error when firstName contains special character" in {
-        val jsonObject: JsValue = Json.parse(
-          """
-            |{
-            | "birthReferenceNumber": "123456789",
-            | "firstName" : "John-->",
-            | "lastName" : "Smith",
-            | "dateOfBirth" : "1997-01-13",
-            | "whereBirthRegistered" : "england"
-            |}
-          """.stripMargin)
-
-        jsonObject.validate[Payload].isError shouldBe true
+        val payload = Payload(None, "-->", None, "Test", LocalDate.now, BirthRegisterCountry.ENGLAND)
+        Json.toJson(payload).validate[Payload].isError shouldBe true
       }
 
       "return error when firstName contains special character |" in {
-        val jsonObject: JsValue = Json.parse(
-          """
-            |{
-            | "firstName" : "|",
-            | "lastName" : "Smith",
-            | "dateOfBirth" : "1997-01-13",
-            | "whereBirthRegistered" : "england"
-            |}
-          """.stripMargin)
-
-        jsonObject.validate[Payload].isError shouldBe true
+        val payload = Payload(None, "|", None, "Test", LocalDate.now, BirthRegisterCountry.ENGLAND)
+        Json.toJson(payload).validate[Payload].isError shouldBe true
       }
 
       "return error when firstName contains newline character" in {
-        val jsonObject: JsValue = Json.parse(
-          """
-            |{
-            | "birthReferenceNumber": "123456789",
-            | "firstName" : "John\n",
-            | "lastName" : "Smith",
-            | "dateOfBirth" : "1997-01-13",
-            | "whereBirthRegistered" : "england"
-            |}
-          """.stripMargin)
-
-        jsonObject.validate[Payload].isError shouldBe true
+        val payload = Payload(None, "John\n", None, "Test", LocalDate.now, BirthRegisterCountry.ENGLAND)
```

Comment:
@chris-ianson What was the decision to move to using the case class in the spec?